src/app/base/components/node/OverviewCard/_index.scss

```diff
@@ -9,33 +9,33 @@
 
     .overview-card__status {
       grid-area: status;
-      padding: $spv--large 0 0 $sph--large;
+      padding: 0.5rem 0 0 $sph--large;
```

Comment:
These should be changed to the appropriate [spacing variable](https://vanillaframework.io/docs/settings/spacing-settings) (in this case probably `$spv--small`).